model/ResultsService.php

```diff
@@ -1312,14 +1317,24 @@ private function convertDates(&$data)
      *
      * @return bool
      */
-    private function filterData($row, array $filters)
+    private function filterData($row, array $filters, string $deliveryExecutionIdentifier)
     {
         $matched = true;
         if (count($filters) && count(array_intersect(self::PERIODS, array_keys($filters)))) {
```

Comment:
I know, that you didn't touch this part, but maybe we can refactor this a bit and use `empty` instead of `count` ?

Just a suggestion to improve performance 😅